### PR TITLE
chore: add changelog

### PR DESCRIPTION
**Pull request by eunjae-lee**
This PR adds `CHANGELOG.md` and a script to automatically update it.

`yarn changelog` will update the `CHANGELOG.md` file, considering the lastly released version (git tag).

`yarn changelog` will be added to the release process later.

Rendered [CHANGELOG.md](https://github.com/algolia/search-insights.js/blob/751036657fe92b886511329baf1407260702dc32/CHANGELOG.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+* **processQueue:** move callback on method level ([#115](https://github.com/algolia/search-insights.js/issues/115)) ([1ff8191](https://github.com/algolia/search-insights.js/commit/1ff8191))
+
+
+
+# [1.2.0](https://github.com/algolia/search-insights.js/compare/v1.1.1...v1.2.0) (2019-04-24)
+
+
+### Bug Fixes
+
+* **processQueue:** extract globalObject from processQueue for easier testing ([#106](https://github.com/algolia/search-insights.js/issues/106)) ([9b56fd9](https://github.com/algolia/search-insights.js/commit/9b56fd9))
```
Comment (tkrugg) [inline CHANGELOG.md]: This one can be removed, it's an internal change, it should have been a chore but I guess I didn't tag it correctly: https://github.com/algolia/search-insights.js/pull/106

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# [1.3.0-beta.0](https://github.com/algolia/search-insights.js/compare/v1.2.0...v1.3.0-beta.0) (2019-09-09)
+
+
+### Bug Fixes
+
+* remove startWith and use indexOf instead ([#129](https://github.com/algolia/search-insights.js/issues/129)) ([9c79e20](https://github.com/algolia/search-insights.js/commit/9c79e20))
```
Comment (tkrugg) [inline CHANGELOG.md]: this one can be tagged **IE 11**

File: package.json [hunk 3]
```diff
@@ -50,6 +51,7 @@
     "babel-preset-env": "1.7.0",
     "body-parser": "1.19.0",
     "bundlesize": "0.18.0",
+    "conventional-changelog-cli": "^2.0.23",
```
Comment (tkrugg) [inline package.json]: let's pin this to follow the convention we follow on the other repos

**Comment by eunjae-lee** [comment]
I've updated the changelog. Let me know how you think about the change.

**Pull request merged**